sources/CMLEXEC

```diff
@@ -1737,26 +1742,28 @@ Copyright (c) 1985-1988, 1990-1991, 1993, 2021 by Venue & Xerox Corporation.
 
 (ADDTOVAR NLAML )
 
-(ADDTOVAR LAMA )
+(ADDTOVAR LAMA PROCESS-EXEC-ID PRINT-EVENT PRINT-HISTORY EXEC-PRIN1 EVENTS-INPUT EVAL-INPUT EXEC-READ)
+)
+(PUTPROPS CMLEXEC COPYRIGHT ("Venue & Xerox Corporation" 1985 1986 1987 1988 1990 1991 1993 2021 2024)
```

Comment:
Is there a way to override the "Venue & Xerox Corporation" attribution to include Interlisp.org?

Reply:
Yes, but it'd probably be better to define a change that we want to apply to ALL of the files that are now under our "control", instead of doing them one at a time.

Reply:
@hjellinek See the discussion with @masinter on my draft PR #1527 from February.

Reply:
Yes, this issue I mentioned in my comment would be superseded by a solution to trust one.